Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -149,6 +149,17 @@ TEST_CASE(test_ilbm_ham6)
     EXPECT_EQ(frame.image->get_pixel(77, 107), Gfx::Color(0xf0, 0x40, 0x40, 0xff));
 }
 
+TEST_CASE(test_ilbm_dos)
+{
+    auto file = MUST(Core::MappedFile::map(TEST_INPUT("ilbm/serenity.lbm"sv)));
+    EXPECT(Gfx::ILBMImageDecoderPlugin::sniff(file->bytes()));
+    auto plugin_decoder = MUST(Gfx::ILBMImageDecoderPlugin::create(file->bytes()));
+
+    auto frame = expect_single_frame_of_size(*plugin_decoder, { 640, 480 });
+
+    EXPECT_EQ(frame.image->get_pixel(315, 134), Gfx::Color(0xff, 0, 0, 0xff));
```

Comment:
`Gfx::Color(0xff, 0, 0, 0xff)` => `Color::NamedColor::Blue`

What are you using to generate/open the image btw?
`https://www.stef.be/dpaint/` does not like it

Reply:
Deluxe Paint 2 DOS version. I guess PBM form files are not supported by this app.

Reply:
That's weird: I had a brief look at DPaint-js source code and PBM files seem to be supported. I'll do some tests/debugging later.

Reply:
It was just added to DPaint-js and is only available in the [dev](https://www.stef.be/dpaint/dev/) version.

The test file I added in this PR can be opened in the dev version.

Reply:
It was just added to DPaint-js and is only available in the [dev](https://www.stef.be/dpaint/dev/) version.

The test file I added in this PR can be opened in the dev version.

Reply:
Thanks for your answer!

Now that I've seen the image, I can confirm that using `NamedColor::Blue` won't work, but using a `NamedColor` would be nice.

And you will need to rebase on master because of #22220.

Reply:
Rebase done.

Reply:
Using `NamedColor::Red` appears to work too.